tests/main/snap-remove-terminate/api-client/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: api-client
+version: 1
+base: core18
```

Comment:
Nitpick, is core18 relevant here? Could we use something less more recent?

Reply:
Good point, I have created ticket (SNAPDENG-26852) to merge the several duplicates of `api-client`, I added a comment about addressing the base there as well.